forumMessages/models.py

```diff
@@ -6,6 +6,16 @@
 class Category(models.Model):
     name = models.TextField(max_length=50)
     # Category.id field auto-generated by Django, will not be used
+    
+    @classmethod
+    def addCategory(self, categoryName):
+        self.categories.add(Category.objects.filter(name=categoryName))
```

Comment:
1. in @classmethod you usually use 'cls' and not 'self' 
2. 'self' is referring to the class you are currently in which is Category and doesn't have a field 'categories'